code/modules/mob/living/simple_animal/hostile/killertomato.dm

```diff
@@ -20,8 +20,18 @@
 	attack_sound = 'sound/weapons/punch1.ogg'
 	ventcrawler = VENTCRAWLER_ALWAYS
 	faction = list("plants")
+	robust_searching = 1
 
 	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 150
 	maxbodytemp = 500
 	gold_core_spawnable = HOSTILE_SPAWN
+
+
+/mob/living/simple_animal/hostile/killertomato/hunter
+	name = "Hunter Tomato"
+	desc = "It's a horrifyingly enormous beef tomato, and it's packing extra beef! This one seems to be a special breed, especially for hunting fauna of all sizes."
+	turns_per_move = 4 //slightly faster
+	maxHealth = 45 //slightly sturdier
+	health = 45
+	friend_types = list("/mob/living/carbon/human")
```

Comment:
```suggestion
	friend_types = list(/mob/living/carbon/human)
```
little bit of coding standards, text strings may cause issues normally unspottable by compiling when handled missing types and the such.
Also they'll end up harming silicons this way.